src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusService.scala

```diff
@@ -1,13 +1,12 @@
 package uk.gov.nationalarchives.tdr.api.service
```

Comment:
Realise its not your code changes but there are `validStatusTypes` and `validStatusValues` vals in the object, could you add a couple of unit tests to make sure these vals are checked? These are pretty crucial so should at least have some testing, until they get moved to a DB table.

Reply:
There's a couple of tests here https://github.com/nationalarchives/tdr-consignment-api/blob/master/src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala#L628 which check the types and values are what we're expecting, is that enough?